devdocs/orientation-guide.md

```diff
@@ -0,0 +1,120 @@
+# **Contributor Orientation Guide**
+
+We welcome contributions from anyone, in particular, students (see [here](https://teammatesv4.appspot.com/about.html) for a list of our contributors). One of the main objectives of TEAMMATES is to help students get experience in a OSS production environment. Here are some information that might be useful to would-be contributors.
+
+
+
+* [Knowledge required](#knowledge-required)
+
+* [Dev Community Structure](#dev-community-structure)
+
+* [Getting started](#getting-started)
+
+* [Contacting us](#contacting-us)
+
+* * *
+
+## Knowledge required
+
+Although TEAMMATES uses many tools and technologies, you need not know all of them before you can contribute. The diagram below shows which technologies you need to learn to contribute in different roles. As you can see, some roles don't need learning any technologies at all. Note that these role aren't fixed or formally assigned. It is simply for guidance only.
+
+![RolesAndTechnologies.png](images/RolesAndTechnologies.png)
+
+Roles:
+
+* **Tester**: Tests the App manually and reports bugs or suggestions for enhancements in the issue tracker.
+*   **Copy editor**: Helps in improving documentation.
+*   **Web page developer**: Works on static web pages, such as those used in the TEAMMATES web site
+*   **Test developer**: Works on automating system tests.
+*   **Frontend developer**: Works on the frontend of the App that is generated dynamically from the server side.
+*   **Backend developer**: Works on the backend logic of the App, including data storage.
+
+## Dev Community Structure
+
+[TEAMMATES community](https://teammatesv4.appspot.com/about.html) has three tiers, defined based on the level of involvement.
+
+*   Contributor: Small but noteworthy contribution to the project.
+*   Committer: Significant contributions sustained over a long period of time.
+*   Core member: Significant and sustained contributions in recent times. Core members can progress through the levels: Snr Developer, Area Lead, Project Lead, and Project Manager.
```

Comment:
You can bold each word "Contributor", "Committer", "Core member" to make it displayed similar to the "Roles" section above.